state_machine/applications/flight/Tasks/deployment_manager.py

```diff
@@ -13,8 +13,12 @@ async def main_task(self):
         if (cubesat.f_contact):
             self.debug('Contact with ground station has been already established, switching to Normal mode')
             state_machine.switch_to('Normal')
+        elif (cubesat.f_burn):
```

Comment:
Is this the behavior we want, or do we want it to attempt another burn after rebooting?